src/main/java/com/kryptnostic/conductor/rpc/odata/Table.java

```diff
@@ -137,13 +138,44 @@ static CassandraTableBuilder createTableDefinition( Table table ) {
                         .ifNotExists()
                         .partitionKey( NAME )
                         .columns( SECURABLE_OBJECTID );
+
+            // TODO: ENTITY_ID( DataType.uuid() ) or ENTITYID( DataType.text() )
+            case AUDIT_LOG:
+                return new CassandraTableBuilder( AUDIT_LOG )
+                        .ifNotExists()
+                        .partitionKey(
+                                CommonColumns.ACL_KEYS,
+                                CommonColumns.EVENT_TYPE
+                        )
+                        .clusteringColumns(
+                                CommonColumns.PRINCIPAL_TYPE,
+                                CommonColumns.PRINCIPAL_ID,
+                                CommonColumns.TIME_UUID
+                        )
+                        .columns(
+                                CommonColumns.ENTITY_ID,
```

Comment:
I think this column is supposed to reference the actual data that got changed. In other words, this column should be a vertexId, or something like that.

Also since we haven't documented this anywhere, I will just leave a brief description of what vertex_id and entity key do. Essentially,
- one row of an entity set is identified by entity_id and sync id. Here entity_id is a string either the user specifies, or we compute in shuttle.
- therefore across all the entity sets, one row of data is uniquely identified by the triple (entity_set_id, entity_id, syncId). We call this an entity key.
- When a row of data is first written to the system, we would register it as a vertex which gives it a unique uuid. In particular, an entity key (hence a row of data) is uniquely identified by the uuid. This is the vertex id we've been talking about.